flink-scala/src/main/scala/org/apache/flink/api/scala/ExecutionEnvironment.scala

```diff
@@ -348,6 +349,47 @@ class ExecutionEnvironment(javaEnv: JavaEnv) {
     wrap(new DataSource[T](javaEnv, inputFormat, typeInfo, getCallLocationName()))
   }
 
+  def readCsvFileAsRow[T : ClassTag : TypeInformation](
+    filePath: String,
+    rowSize: Int,
+    additionalTypes: Map[Int, Class[_]] = null,
+    lineDelimiter: String = "\n",
+    fieldDelimiter: String = ",",
+    quoteCharacter: Character = null,
+    ignoreFirstLine: Boolean = false,
+    ignoreComments: String = null,
+    lenient: Boolean = false,
+    includedFields: Array[Int] = null): DataSet[Row] = {
```

Comment:
Please add more two spase for parameters like as for other methods in class, space chars should is 6.